client/ionic/package.json

```diff
@@ -29,7 +29,14 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "check": "gts check",
+    "clean": "gts clean",
```

Comment:
possible collision/confusion with npm's `clean` - not sure if we care enough to rename or not

Reply:
renamed to tsclean for now